test/Crate/Test/DBAL/Platforms/CratePlatformTest.php

```diff
@@ -74,23 +84,133 @@ public function getGenerateAlterTableSql()
             'ALTER TABLE mytable ADD quota INTEGER',
         );
     }
+    
+    public function testAlterTableChangeQuotedColumn()
+    {
+        $this->markTestSkipped('Platform does not support ALTER TABLE.');
+    }
 
     protected function getQuotedColumnInPrimaryKeySQL()
     {
         return array(
-            'CREATE TABLE "quoted" ("key" STRING, PRIMARY KEY("key"))',
+            'CREATE TABLE "quoted" ("create" STRING, PRIMARY KEY("create"))',
         );
     }
 
     protected function getQuotedColumnInIndexSQL()
     {
         return array(
-            'CREATE TABLE "quoted" ("key" STRING,' .
-            'INDEX IDX_22660D028A90ABA9 USING FULLTEXT ("key")' .
+            'CREATE TABLE "quoted" ("create" STRING, ' .
+            'INDEX IDX_22660D028FD6E0FB USING FULLTEXT ("create")' .
             ')'
         );
     }
 
+    /**
+     * @todo
+     */
+    protected function getQuotedNameInIndexSQL()
+    {
+        return array(
+            'CREATE TABLE test (column1 STRING, INDEX key USING FULLTEXT (column1))'
+        );
+    }
+    
+    /**
+     * @todo
+     */
+    protected function getQuotedColumnInForeignKeySQL()
+    {
+        $this->markTestSkipped('Platform does not support ADD FOREIGN KEY.');
```

Comment:
Ok, i've done it because there is many places in crate tests that you've done similar thing:
 * https://github.com/crate/crate-dbal/blob/master/test/Crate/Test/DBAL/Platforms/CratePlatformTest.php#L58
 * https://github.com/crate/crate-dbal/blob/master/test/Crate/Test/DBAL/Platforms/CratePlatformTest.php#L63

... and more.

But you're right -  `skip test` methods need to be moved and generators to removed.